packages/react/src/components/buttons/abstract/abstract-button.tsx

```diff
@@ -9,18 +9,29 @@ export const AbstractButton = forwardRef<HTMLButtonElement, PropsWithChildren<Ab
     focusable,
     isMobile: providedIsMobile,
     size,
+    disabled,
     ...props
 }: AbstractButtonProps, ref: Ref<HTMLButtonElement>) => {
     const { isMobile } = useDeviceContext();
 
+    const handleClick = (event: MouseEvent<HTMLButtonElement>): void => {
+        if (disabled) {
+            event.preventDefault();
+            event.stopPropagation();
+        } else if (onClick) {
+            onClick(event);
+        }
+    };
```

Comment:
Peut-être wrapper dans un `useCallback`
```suggestion
    const handleClick = useCallback((event: MouseEvent<HTMLButtonElement>): void => {
        if (disabled) {
            event.preventDefault();
            event.stopPropagation();
        } else if (onClick) {
            onClick(event);
        }
    }, [disabled, onClick]);
```